onnxscript/tests/function_libs/torch_lib/error_reproduction.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations
+
+import pathlib
+import platform
+import sys
+import time
+import traceback
+from typing import Any, Mapping
+
+import numpy as np
+import onnx
+import onnxruntime as ort
+import torch
+
+_REPRODUCTION_TEMPLATE = '''\
+import google.protobuf.text_format
+import numpy as np
+from numpy import array, float16, float32, float64, int32, int64
+import onnx
+import onnxruntime as ort
+
+# Run n times
+N = 1
+
+onnx_model_text = """
+{onnx_model_text}
```

Comment:
I can see this is more convenient that we have a single repro file. I'm slightly worried when there are large initializers in the model. Perhaps it is fine since this is for op level unittests. Maybe add your considerations on this as NOTE comment.

Reply:
Will do.

Reply:
Done